src/App.tsx

```diff
@@ -1,25 +1,53 @@
-import React from 'react';
+import React, { ChangeEvent, useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42).map(n => `Item ${n}`);
+const items : string[] = getNumbers(1, 42).map(n => `Item ${n}`);
+
+enum PerPageOption {
+  'three' = 3,
+  'five' = 5,
+  'ten' = 10,
+  'twenty' = 20
+}
 
 export const App: React.FC = () => {
+  const [perPageSelector, setPerPageSelector] = useState<PerPageOption>(PerPageOption.five);
+  const [activePage, setActivePage] = useState<number>(1);
+
+  function handleSelect (e: ChangeEvent<HTMLSelectElement>) {
+    setPerPageSelector(+e.currentTarget.value)
```

Comment:
The unary plus operator is used to convert the string value to a number. While this works, it might be clearer to use `parseInt(e.currentTarget.value, 10)` to explicitly convert the string to an integer.